Tests/TestingTests/Traits/CustomTraitTests.swift

```diff
@@ -8,11 +8,75 @@
 // See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 //
 
-// Note: Do NOT include `@_spi` on this import. The contents of this file are
-// specifically intended to validate that a type conforming to `TestTrait` can
-// be declared without importing anything except the base.
-import Testing
+@testable @_spi(ExperimentalTestRunning) @_spi(ExperimentalEventHandling) import Testing
```

Comment:
I had this comment here with the goal that we would maintain a test file which does _not_ include any `@_spi` attributes on an import of `Testing` module, because that's important to validate that a custom `Trait` type can be declared without needing SPIs. We had a bug in the past about this, and I'd like to ensure we don't lose our ability to test that.

Can we move your newly-added test to a distinct file, so that this `@_spi`-less import can be preserved

Reply:
Oh, this PR has been open for too long - I wrongly assumed that I had added it and wasn't sure if it would be a valuable addition but yes, I'll re-add it and move my tests into another file.